test/cypress/integration/settings/your-psp.cy.test.js

```diff
@@ -78,13 +83,20 @@ describe('Your PSP settings page', () => {
       issuer: testFailureFlexCredentials.issuer,
       jwt_mac_key: testFailureFlexCredentials.jwt_mac_key
     })
+    const postCheckWorldpay3dsFlexCredentialsReturnsBadResult = gatewayAccountStubs.postCheckWorldpay3dsFlexCredentialsWithBadResult({
+      gatewayAccountId: gatewayAccountId,
+      organisational_unit_id: testBadResultFlexCredentials.organisational_unit_id,
```

Comment:
If you wanted to you could use the `...` operator on the `testBadResultFlexCredentials` object rather than having to reference each of its attributes individually, like
```
> { gatewayAccountId: "someid", ...testBadResultFlexCredentials }
{
  gatewayAccountId: 'someid',
  organisational_unit_id: '5bd9b55e4444761ac0af1c83',
  issuer: '5bd9e0e4444dce153428c943',
  jwt_mac_key: 'fa2daee2-1fbb-45ff-4444-52805d5cd9e3'
}
```

Just thought I'd mention it in passing, since you posted the PR in the Kaizen channel I'll let someone from your team review the PR, looks ok to me though 👍

Reply:
Thanks @danworth. I have made the changes.